tools/README.md

```diff
@@ -47,7 +42,7 @@ images are:
 The script [delete_prebuilt_workers.go](prebuilt_workers/delete_prebuilt_workers.go) 
 deletes images within user specified registry. The script lists all images
 within the specified registry, then check if the image has the user specified 
-tag.
+tag. The script only support Google cloud registry so far.
```

Comment:
A couple comments:

- We don't have any intention to support other registries with this script right now, and we're not really looking for PRs with them. Therefore, I'd just leave off the "so far" phrase. 

- The official name of the product is "Google Container Registry," so let's just use that. You can add that the limitation is due to depending on the google-cloud-sdk. 
 
```suggestion
tag. This script only supports Google Container Registry, because it relies on the google-cloud-sdk.
```